src/Package/ConfigInitializer.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace SiteStudio\Package;
+
+class ConfigInitializer {
+
+    /**
+     * Src Package.
+     *
+     * @var string
+     */
+    protected $srcPackageRepo;
+
+    /**
+     * Target Package.
+     *
+     * @var string
+     */
+    protected $tarPackageRepo;
+
+    /**
+     * Src Branch or Tag.
+     *
+     * @var string
+     */
+    protected $srcBranchTag;
+
+    /**
+     * Target branch or tag.
+     *
+     * @var string
+     */
+    protected $tarBranchTag;
+
+    /**
+     * GitLab API Path.
+     *
+     * @var string
+     */
+    protected $gitlabApiPath = "https://gitlab.bayer.com/api/v4";
+
+    /**
+     * Package Repo Group.
+     *
+     * @var string
+     */
+    protected $packageRepoGroup;
+
+    /**
+     * Comparision Folder.
+     *
+     * @var string
+     */
+    protected $comparisionFolder = 'package-comparision';
+
+    /**
+     * Projects Details.
+     *
+     * @var string
+     */
+    protected $projectsPath = "projects.json";
```

Comment:
@rakshithtb nitpick - I'd provide more descriptive information - what is this file, is it autogenerated?